pkg/virt-controller/watch/migration.go

```diff
@@ -504,6 +521,8 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 	controller.SetVMIMigrationPhaseTransitionTimestamp(migration, migrationCopy)
 	controller.SetSourcePod(migrationCopy, vmi, c.podIndexer)
 
+	c.flushPendingQueue(migration.Status.Phase, migrationCopy.Status.Phase)
```

Comment:
Should we perform this after the `UpdateStatus` is performed?
I mean, if the update status fails there is the possibility that the migrations we have enqueued will be pushed back to the pending queue. No?